package.json

```diff
@@ -186,7 +196,7 @@
 				"program": "./out/src/debug/debug_entry.js",
 				"runtime": "node",
 				"variables": {
-					"sdkPath": "dart.getSdkPath"
+					"debugSettings": "dart.getDebugSettings"
```

Comment:
Yes :(

I haven't thoroughly tested this yet, but I think when you hit `F5` Code notices the values mismatch and tell you to fix it up (and open the file), though it's not very obvious. I don't know if there's any good way to handle this (besides calling out in release notes) since we can't hook `F5`.

Possibly at startup we could detect the badness and show an info tip with better information (and maybe a link to the release notes that describe it in detail)?